bsp/src/org/jetbrains/bsp/project/importing/BspResolverLogic.scala

```diff
@@ -738,9 +754,10 @@ private[importing] object BspResolverLogic {
 
       case SbtModule(_, scalaSdkData, sbtData) =>
         val scalaSdkNode = new DataNode[ScalaSdkData](ScalaSdkData.Key, scalaSdkData, moduleNode)
-        val sbtNode = new DataNode[SbtBuildModuleDataBsp](SbtBuildModuleDataBsp.Key, sbtData, moduleNode)
+        val sbtBuildModuleDataNode = new DataNode[SbtBuildModuleDataBsp](SbtBuildModuleDataBsp.Key, sbtData, moduleNode)
         moduleNode.addChild(scalaSdkNode)
-        moduleNode.addChild(sbtNode)
+        moduleNode.addChild(sbtBuildModuleDataNode)
+        moduleNode.addChild(new SbtModuleNode(SbtModuleData(moduleData.id, new URI(moduleData.id))).toDataNode)
```

Comment:
I didn't really know what I was doing in this part of the patch, so worth double checking if it makes sense.

Reply:
oh yeah that's using a class from the sbt support, which it shouldn't in the BSP context :D

Reply:
Maybe there is an argument that its okay to have this dependency when the SBT extension is used. This lets us share the project settings UI.

Reply:
Okay, let's leave this redundancy in for now, though I hope to eliminate it when unifying BSP and sbt. My idea is to make BSP the default sbt mechanism for newer sbt versions, transparently to the user.